### DISCUSSION TITLE
Has any pure use nicegui to build UI components?

### DISCUSSION THREAD

**Discussion by italanchan** [zauberzeug/nicegui#1925] [Q&A]
### Question

I want some samples codes to build UI components use only nicegui framework as js and html is not my strengths.

Now, a Stat box component is plan to build use Nicegui.  

example.
https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/#big-numbers--stats

**Comment by rodja** [comment]
NiceGUI currently provides `ui.highcharts` and `ui.echart`. Maybe one of them suits your needs? You'll need to look in their project documentation to find the right parameters.
